tests/test_mail_sink.py

```
@@ -1,9 +1,15 @@
+import re
+from email import message_from_string
+from html import unescape
 from io import BytesIO
+from typing import Dict, List
 from unittest.mock import ANY, call, patch
+from urllib.parse import parse_qs, urlparse
 
 import pytest
 from apprise.attachment import AttachFile
```

Flake8 (7.3.0)

[error] 'apprise.attachment.AttachFile' imported but unused

(F401)